Day-11-Quiz-App/index.html

```diff
@@ -113,13 +119,90 @@ <h2>Custom Quizzes (Work in progress)</h2>
             customQuizContainer.style.display = "none";
 
         });
+
         quizElement.addEventListener("QuizBuilder:quit", () => {
+            // TODO: pass data of quiz that was just saved along in event, so that we only need to add one more button for the new quiz instead of rebuilding all buttons.
             quizChooser.style.display = "block";
             customQuizContainer.style.display = "block";
+            // TODO: implement dedicated QuizBuilder:save event, as we do not need to rebuild the buttons if no quiz was created.
+            displayCustomQuizButtons(getCustomQuizzesFromLocalStorage()); // reloads custom quiz buttons
+            displaySelectForEditingAndDeletingCustomQuizzes(getCustomQuizzesFromLocalStorage());

```

Comment:
The following snippet is duplicated three times. Please extract it into a function with a meaningful name and call that function instead.
```javascript
displayCustomQuizButtons(getCustomQuizzesFromLocalStorage());
displaySelectForEditingAndDeletingCustomQuizzes(getCustomQuizzesFromLocalStorage());
```

Reply:
done in https://github.com/sik-leanix/100-days-of-javascript/pull/2/commits/31920d7f8957059b160a9ad57cb66a46923dff76